src/powerflow_types.jl

```diff
@@ -1,6 +1,10 @@
 abstract type PowerFlowEvaluationModel end
 
-Base.@kwdef struct ACPowerFlow <: PowerFlowEvaluationModel
+Base.@kwdef struct NLSolveACPowerFlow <: PowerFlowEvaluationModel
+    check_reactive_power_limits::Bool = false
+end
+
+Base.@kwdef struct KLUACPowerFlow <: PowerFlowEvaluationModel
```

Comment:
Comment from José: go back to one struct for all AC power flow, specify the algorithm with struct parameter or field or something